<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>345</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>345</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Animation manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>mpl_animationmanager/images/icon.png</normaloff>mpl_animationmanager/images/icon.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>dpi:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_7">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>80</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_dpi">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>80</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>fps:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="value">
          <number>24</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_fps">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_modif" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QCheckBox" name="checkBox_modif">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Modification period, s:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_5">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_period_modif">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_rot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBox_rot">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Rotation period, s:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_2">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_period_rot">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Elevation:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_3">
          <property name="maximum">
           <number>90</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_elev">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Initial azimut:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_4">
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>-30</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_azim">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>-30</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>mpl_animationmanager/images/play.png</normaloff>mpl_animationmanager/images/play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPause">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>mpl_animationmanager/images/pause.png</normaloff>mpl_animationmanager/images/pause.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>mpl_animationmanager/images/stop.png</normaloff>mpl_animationmanager/images/stop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>File name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>my_animation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowse">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Writer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_writers"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_ext">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>mp4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gif</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAsk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>mpl_animationmanager/images/Info.png</normaloff>mpl_animationmanager/images/Info.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnShow">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>Export</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>widget_rot</zorder>
  <zorder>widget_modif</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_fps</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_fps</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_period_rot</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_period_rot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_elev</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_elev</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_azim</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_azim</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_period_modif</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_period_modif</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_modif</sender>
   <signal>toggled(bool)</signal>
   <receiver>horizontalSlider_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_modif</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_period_modif</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_modif</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_10</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_rot</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_period_rot</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_rot</sender>
   <signal>toggled(bool)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_rot</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>12</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_dpi</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_dpi</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
